sentry-android-core/src/main/java/io/sentry/android/core/SpotlightIntegration.java

```diff
@@ -0,0 +1,122 @@
+package io.sentry.android.core;
+
+import static io.sentry.SentryLevel.DEBUG;
+import static io.sentry.SentryLevel.ERROR;
+
+import io.sentry.Hint;
+import io.sentry.IHub;
+import io.sentry.Integration;
+import io.sentry.SentryEnvelope;
+import io.sentry.SentryOptions;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.zip.GZIPOutputStream;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class SpotlightIntegration
```

Comment:
Could we also build this in a way that works for backend, desktop, etc. as well?

Reply:
:+1: I just moved it over to sentry core.

I couldn't get the spring-boot example running though, as it somehow didn't pick up my config. Maybe you can have a quick look? 

It's easiest to get started by using the pre-built spotlight electron app (https://spotlightjs.com/) +
setting `options.setSpotlightConnectionUrl("http://localhost:8969/stream");`

Reply:
Will give it a try.